### ISSUE TITLE
Sign-up: What does your business do? lacks label for form field

### ISSUE THREAD

**Issue by davidakennedy** [Automattic/wp-calypso#32704]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/start/
2. Select Business and proceed to next step.
3. The field for "What does your business do?" has no associated label. See: https://wordpress.com/start/onboarding-for-business/site-topic-with-preview

#### What I expected

I expect the form to have a label.

#### What happened instead

The form has no label, so a screen reader or other assistive technology can't read it.

#### Browser / OS version

MacOS / Chrome.

#### Screenshot / Video

![Screen Shot 2019-04-30 at 5 23 01 PM](https://user-images.githubusercontent.com/1473618/56994631-9d872d80-6b6d-11e9-8e7c-d96b4ac61028.png)

#### Fix

I would give the "What does your business do?" heading an `id` and associate the form field with it. That way, no design tweaks are needed, and no off-screen labels are needed.

#### Context / Source

#manual-testing

**Comment by davidakennedy** [comment]
Same issue found here for the site name here: https://wordpress.com/start/onboarding-for-business/site-title-with-preview

**Comment by alaczek** [comment]
@davidakennedy would it be acceptable to use `aria-label` on the input itself ([as mentioned here](https://www.w3.org/WAI/tutorials/forms/labels/)) instead? Looking at the code, the header is a reusable component, that doesn't support `id` for the heading (`h1`) element.

**Comment by alaczek** [comment]
> Same issue found here for the site name here: https://wordpress.com/start/onboarding-for-business/site-title-with-preview

Correct me if I'm wrong but the site title step does have a label:
![image](https://user-images.githubusercontent.com/4550351/57495420-5a694f00-7311-11e9-90d4-991b46cfc6a6.png)

Although it's not what heading above says ("Tell us your site's name" versus "What is the name of your business?").

**Comment by alaczek** [comment]
This is fixed now, by adding `aria-label` to inputs, and removing unused `<label>`
